### ISSUE TITLE
dag mode - text and node looks overlapped

### ISSUE THREAD

**Issue by ravik3592** [vasturiano/force-graph#85]
I tried to render data in dag td mode but the data is not readable. Can you suggest any options to make it look better and readable
![dag](https://user-images.githubusercontent.com/56120798/72057500-856acf00-32f4-11ea-92e5-32e8303c8372.PNG)

**Comment by vasturiano** [comment]
Making it horizontal (`lr`) will prevent  the node texts from overlapping as much.

**Comment by ravik3592** [comment]
yes making it lr worked but is it possible to make text rotate in this type of tree

**Comment by vasturiano** [comment]
@ravik3592 since you are drawing the nodes with `nodeCanvasObject` you are in full control on how the labels are drawn. So you can apply some rotation to them inside that method.

**Comment by ravik3592** [comment]
Is there an option to increase node spacing when texts overlaps

**Comment by manunamz** [comment]
Linking relevant [issue](https://github.com/vasturiano/force-graph/issues/222).
